### DISCUSSION TITLE
Cannot read property 'slice' of undefined pls help

### DISCUSSION THREAD

**Discussion by bdloner** [strapi/strapi#6113] [Legacy Topics]
![image](https://user-images.githubusercontent.com/15920037/81326368-edbf9680-90c3-11ea-9e3e-e22aaff4c134.png)

![image](https://user-images.githubusercontent.com/15920037/81326409-fc0db280-90c3-11ea-9ae6-d33058c2dca1.png)

![image](https://user-images.githubusercontent.com/15920037/81326554-37a87c80-90c4-11ea-8eb4-33f9e56622e1.png)

need to fix this 

thank you

**Comment by robertpiosik** [comment]
You didn't get the API response due to wrong, as error said, URL. An absolute URL is the form of URL that contains protocol and domain name, eg. `https://example.com/my-api-endpoint`

**Reply by bdloner** [reply]
I just cloned a github and build. Now I don't know where to fix the end point.

**Reply by robertpiosik** [reply]
[This one](https://strapi.io/blog/strapi-starter-vue-blog)? It works on my end.

**Reply by bdloner** [reply]
yes, but not work for me 😭

**Reply by robertpiosik** [reply]
Try to delete the folder, clone repo again, install deps again and see if it helps. Look carefully which commands do you use.

> yarn setup:yarn
> yarn build:yarn  
> yarn develop:yarn

do the trick

**Reply by bdloner** [reply]
yeah it's work thank you very much 😄
